backend/grant/proposal/commands.py

```diff
@@ -90,36 +90,79 @@ def create_proposals(count):
 @with_appcontext
 def retire_v1_proposals(dry):
     now = datetime.datetime.now()
-    proposals = Proposal.query.filter_by(stage="FUNDING_REQUIRED").all()
-    modified_count = 0
+    proposals_funding_required = Proposal.query.filter_by(stage="FUNDING_REQUIRED").all()
+    proposals_draft = Proposal.query.filter_by(status=ProposalStatus.DRAFT).all()
+    proposals_pending = Proposal.query.filter_by(status=ProposalStatus.PENDING).all()
+    modified_funding_required_count = 0
+    modified_draft_count = 0
+    modified_pending_count = 0
 
-    if not proposals:
+    # TODO: what about APPROVED (before use publishes the proposal live) or STAKING stages?
+    # TODO: what happens to PENDING proposals that have already staked?
```

Comment:
These are just proposals in a pending state, so we should be fine?

Reply:
same goes here, they should be sure to get into a "v2" state.

Reply:
I was curious if these proposals would have to stake again, but this might not be the case since rejected and resubmitted proposals don't have to stake a second time.

Reply:
and we're removing staking anyway, so they definitely won't need to.